### PR TITLE
Demo tp

### PR DESCRIPTION
**Pull request by jmunozv**
First version of DEMO++ TrackingPlane implementing both the "original" one, and the "type1" currently in use.
The two types of TrackingPlane are selectable by parameter.
Both of them make use of the new NextDemoSiPMBoard.

### CODE REVIEW

File: source/geometries/NextDemoSiPMBoard.cc [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+                        mask_name);
+
+  // Adding the optical surface
+  G4OpticalSurface* mask_opsurf = 
+    new G4OpticalSurface(mask_name, unified, ground, dielectric_metal);
+  mask_opsurf->SetMaterialPropertiesTable(OpticalMaterialProperties::PTFE());
+  new G4LogicalSkinSurface(mask_name + "_OPSURF", mask_logic, mask_opsurf);
+
+  new G4PVPlacement(nullptr, G4ThreeVector(0., 0., mask_posz), mask_logic,
+                    mask_name, board_logic, false, 0, true);
```
Comment (paolafer) [inline source/geometries/NextDemoSiPMBoard.cc:166]: Before approving this PR, please set to false all the overlap checks.

File: source/geometries/NextDemoTrackingPlane1.cc [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  delete msg_;
+  delete sipm_board_;
+  delete plate_gen_;
+}
+
+
+
+void NextDemoTrackingPlane1::Construct()
+{
+  /// Verbosity
```
Comment (paolafer) [inline source/geometries/NextDemoTrackingPlane1.cc]: Remove comments or uncomment.

**Review by paolafer — CHANGES_REQUESTED** [review]

File: source/geometries/NextDemoTrackingPlane1.cc (hunk 2 above)
Comment (jmunozv) [reply source/geometries/NextDemoTrackingPlane1.cc]: Done

**Comment by paolafer** [comment]
The variable `gate_tp_copper_distance_` is not used in NextDemoInnerElements, please remove it.

**Comment by jmunozv** [comment]
> The variable `gate_tp_copper_distance_` is not used in NextDemoInnerElements, please remove it.

Done

File: source/geometries/NextNewKDB.cc [hunk 3]
```diff
@@ -61,7 +61,7 @@ namespace nexus {
```
Comment (jmalbos) [inline source/geometries/NextNewKDB.cc:61]: This change in a NEW file shouldn't have been committed.

Comment (jmunozv) [reply source/geometries/NextNewKDB.cc:61]: My mistake. Sorry.

**Review by paolafer — APPROVED** [review]
The geometry looks fine, I'm going to approve this PR.

**Pull request merged**